rust/auction/program/src/processor/claim_bid.rs

```diff
@@ -57,13 +58,15 @@ fn parse_accounts<'a, 'b: 'a>(
         bidder_pot: next_account_info(account_iter)?,
         authority: next_account_info(account_iter)?,
         auction: next_account_info(account_iter)?,
+        auction_extended: next_account_info(account_iter)?,
```

Comment:
So this is a problem. When adding to an existing contract endpoint, you can't put an account in the middle of the expected set because you will break existing older client UIs. So in this instance what you can do is put it beneath token program, and make it optional. If it's not present in the next_account_info, then you wont check for instant sale-ness, and the caller must wait till the auction ends like normal. If they wish to claim early, they need to provide auction data extended.